### PR TITLE
5.191 Release

### PR DESCRIPTION
**Pull request by Pandrex247**
Release Docs

### CODE REVIEW

File: release-notes/release-notes-191.adoc [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+- Executor Service
+- Async CDI events
+- EJB Timers
+- Security
+- Access log
+
+If you are uncertain if this might impact your application, please feel free to ask more information.
+
+There are new as asadmin commands for the configuration of the health check feature, the old ones are still valid but are deprecated.
```
Comment (OndroMih) [inline release-notes/release-notes-191.adoc]: "new as asadmin" - remove "as".

File: release-notes/release-notes-191.adoc [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+-   https://github.com/payara/Payara/pull/3728[PAYARA-3457] - Allow disabling auto-increment of Hazelcast port selection for the Domain Data Grid
+-   https://github.com/payara/Payara/pull/3704[PAYARA-3458] - Improve Domain Data Grid Startup/Status Log Output
+-   https://github.com/payara/Payara/pull/3761[PAYARA-3473] - Make Hazelcast initial wait time configurable
+-   https://github.com/payara/Payara/pull/3749[PAYARA-3499] - Optimise ASClassLoaderUtil
+-   https://github.com/payara/Payara/pull/3757[PAYARA-3505] - Prevent JLine Logging in Asadmin Multimode
+-   https://github.com/payara/Payara/pull/3759[PAYARA-3507] - Improve the Executor Pool and Queue Sizes in Payara Micro
+-   https://github.com/payara/Payara/pull/3760[PAYARA-3508] - Improve logging in ConnectionPool
+-   https://github.com/payara/Payara/pull/3737[PAYARA-3514] - ***Community Contribution*** Typo and newline fix
+
+== Security
```
Comment (OndroMih) [inline release-notes/release-notes-191.adoc]: rather "Security Fixes"

**Comment by Pandrex247** [comment]
Added 4.191 release notes

**Comment by Pandrex247** [comment]
Yes they came from the 5.184 branch.

**Pull request merged**